lib/puppet/type/group.rb

```diff
@@ -80,7 +80,9 @@ def sync
 
     newproperty(:members, :array_matching => :all, :required_features => :manages_members) do
       desc "The members of the group. For directory services where group
```

Comment:
The point of that is that there are groups and users and the only difference is that one is shown as a group.